### DISCUSSION TITLE
Import everything fails

### DISCUSSION THREAD

**Discussion by labonny** [requarks/wiki#7298] [Help / Questions]
I'm running a new installation of Wiki.js (v 2.5.303) using docker compose on a Rocky Linux 9.3 host. I am trying to import a directory of html files. I've mounted this directory into the containers (both db & wiki), and have checked that it is readable from both containers.

I tried to import the pages using: Storage / Local File System /  Import Everything, but no pages get created. When using docker container logs -f <wiki_container_name> to see the logs during import, I can see lots of warnings like this:
2024-08-12T20:33:07.588Z [MASTER] warn: EISDIR: illegal operation on a directory, read
2024-08-12T20:33:07.588Z [MASTER] info: (STORAGE/DISK) Processing wiki/node_modules/zen-observable...
2024-08-12T20:33:07.588Z [MASTER] info: (STORAGE/DISK) Asset marked for import: wiki/node_modules/zen-observable
2024-08-12T20:33:07.597Z [MASTER] warn: EISDIR: illegal operation on a directory, read

The process runs for several minutes, with no error output on the web page, but no wiki pages are visible after it is complete.

Is there some expected directory structure for the directory of pages to import? It is just a flat directory of html files.

**Comment by prohtex** [comment]
> I'm running a new installation of Wiki.js (v 2.5.303) using docker compose on a Rocky Linux 9.3 host. I am trying to import a directory of html files. I've mounted this directory into the containers (both db & wiki), and have checked that it is readable from both containers.
> 
> I tried to import the pages using: Storage / Local File System / Import Everything, but no pages get created. When using docker container logs -f <wiki_container_name> to see the logs during import, I can see lots of warnings like this: 2024-08-12T20:33:07.588Z [MASTER] warn: EISDIR: illegal operation on a directory, read 2024-08-12T20:33:07.588Z [MASTER] info: (STORAGE/DISK) Processing wiki/node_modules/zen-observable... 2024-08-12T20:33:07.588Z [MASTER] info: (STORAGE/DISK) Asset marked for import: wiki/node_modules/zen-observable 2024-08-12T20:33:07.597Z [MASTER] warn: EISDIR: illegal operation on a directory, read
> 
> The process runs for several minutes, with no error output on the web page, but no wiki pages are visible after it is complete.
> 
> Is there some expected directory structure for the directory of pages to import? It is just a flat directory of html files.

Great question. I have the same problem. Did you get anywhere with it?

**Reply by labonny** [reply]
No, I didn't have time to continue troubleshooting this, sorry

**Reply by ReySadeghi** [reply]
I have the same problem, if any solution found please share here.
